content/articles/how-to-post-form-data-as-json-using-the-formdata-web-api-and-fetch-api/index.md

```diff
@@ -0,0 +1,362 @@
+### How To Use The FormData and Fetch API to Submit Data As JSON Data To An API Endpoint
+
+The browser [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) is used to make requests from a web page on the front end to an API endpoint on the back end. The [FormData API](https://developer.mozilla.org/en-US/docs/Web/API/FormData) on the other hand provides a precise way of accessing HTML form fields. Both of these two natively supported browser APIs make it easy to to format and send HTTP requests, but to send data as JSON to an API endpoint extra bit of work is needed. In this tutorial, we explore a step by step procedure on how to capture form field data, format them as JSON data and send it to an API endpoint.
+
+### Steps
+
+Here are the steps we are going to follow to achieve our goal
+
+1. [Listen for form submission](#Listen-for-form-submission).
+2. [Reading the form field values with the browser FormData API](#reading-the-form-field-values-with-the-browser-formdata-api).
+3. [Formatting the data as JSON](#formatting-the-data-as-json)
+4. [POST the JSON data to a URL endpoint with Fetch API as request body data](#post-the-json-data-to-a-url-endpoint-with-fetch-api-as-request-body-data).
+5. [Handling JSON request body with Express Nodejs API](#handling-json-request-body-with-express-nodejs-api).
+
+### Goal
+
+By the end of this tutorial, we will have a working HTML form that sends form data as JSON to an API endpoint. In the process, we will make use of the native browser Fetch and FormData APIs to achieve our goal. As a bonus, we have a simple Express Nodejs server which will listen for the request we send and send back a response.
+
+### Prerequisites
+
+It’s assumed you have prior experience with HTML, CSS and JS.
```

Comment:
Your "Prerequisites" should be listed, please, stating whether a beginner knowledge is needed or not.